src/platform/STM32/platform_mcu.h

```diff
@@ -385,8 +385,8 @@ extern uint8_t _dmaram_end__;
 #define MAX_SPI_PIN_SEL 5
 #elif defined(STM32H5)
 /// [Project-H5] source: https://www.st.com/resource/en/datasheet/stm32h562ag.pdf | version: DS14258 Rev 5 | pages 57
-// note: there is 6 SPI and 3 of them (1,2,3) are I2S compatible
-#define MAX_SPI_PIN_SEL 6
+// note: there is 3 multiplexed SPI (1,2,3)
```

Comment:
This is same as uart - maximum number of multiplexing combination available (used in BF) for sck/miso/mosi